cmd/compute-domain-kubelet-plugin/driver.go

```diff
@@ -108,28 +111,28 @@ func (d *driver) Shutdown() error {
 	if err := d.state.computeDomainManager.Stop(); err != nil {
 		return fmt.Errorf("error stopping ComputeDomainManager: %w", err)
 	}
-	d.plugin.Stop()
+	d.helper.Stop()
 	return nil
 }
 
-func (d *driver) NodePrepareResources(ctx context.Context, req *drapbv1.NodePrepareResourcesRequest) (*drapbv1.NodePrepareResourcesResponse, error) {
-	klog.Infof("NodePrepareResource is called: number of claims: %d", len(req.Claims))
-	preparedResources := &drapbv1.NodePrepareResourcesResponse{Claims: map[string]*drapbv1.NodePrepareResourceResponse{}}
+func (d *driver) PrepareResourceClaims(ctx context.Context, claims []*resourceapi.ResourceClaim) (map[types.UID]kubeletplugin.PrepareResult, error) {
+	klog.Infof("PrepareResourceClaims called with %d claim(s)", len(claims))
 
 	var wg sync.WaitGroup
 	ctx, cancel := context.WithTimeout(ctx, ErrorRetryMaxTimeout)
 	workQueue := workqueue.New(workqueue.DefaultControllerRateLimiter())
+	results := make(map[types.UID]kubeletplugin.PrepareResult)
```

Comment:
see https://github.com/kubernetes/dynamic-resource-allocation/blob/v0.33.0/kubeletplugin/draplugin.go#L119 for that interface

Reply:
Does renaming `results` to `resultsByClaim` aid in readability?

Reply:
Good question. I think in this case not really.